From: Alexander Neundorf <neundorf@kde.org>
From: Modestas Vainius <modax@debian.org>
Subject: don't set the install RPATH to LIB_INSTALL_DIR
 don't set the install RPATH to LIB_INSTALL_DIR if LIB_INSTALL_DIR is one of
 the standard system link directories, like e.g. /usr/lib (which are listed in
 CMAKE_PLATFORM_IMPLICIT_LINK_DIRECTORIES, which is set in
 Modules/Platform/UnixPaths.cmake).
 .
 Also drop custom INSTALL_RPATHs in kdewidgets/CMakeLists.txt.
Forwarded: yes
Origin: backport, http://websvn.kde.org/?view=revision&revision=1124215
Last-Update: 2010-06-23

Index: kde4libs/kdewidgets/CMakeLists.txt
===================================================================
--- kde4libs.orig/kdewidgets/CMakeLists.txt	2010-11-05 11:48:27.000000000 +0100
+++ kde4libs/kdewidgets/CMakeLists.txt	2010-11-25 16:23:01.000000000 +0100
@@ -51,7 +51,6 @@
                                      INSTALL_RPATH_USE_LINK_PATH TRUE
                                      SKIP_BUILD_RPATH TRUE
                                      BUILD_WITH_INSTALL_RPATH TRUE
-                                     INSTALL_RPATH ${LIB_INSTALL_DIR}
                          )
   endif(NOT WIN32)
 
@@ -152,7 +151,6 @@
                                      INSTALL_RPATH_USE_LINK_PATH TRUE
                                      SKIP_BUILD_RPATH TRUE
                                      BUILD_WITH_INSTALL_RPATH TRUE
-                                     INSTALL_RPATH ${LIB_INSTALL_DIR}
                              )
      endif(NOT WIN32)
 
